app.js

```diff
@@ -0,0 +1,39 @@
+
+function buildContactList() {
+  const allAddress = document.getElementById('contact_details');
+  return Array.from(allAddress.rows);
+}
+
+function infoMessage(toBeDeleted) {
+  const note = document.getElementById('error_message');
+
+    if(toBeDeleted === undefined) {
```

Comment:
something happened with the indentation :slightly_smiling_face: 
remember, select the text and `Ctrl` + `i`